doc/sphinx/source/graph-basics.rst

```diff
@@ -68,9 +164,9 @@ and results in a tensor with the same element type and shape:
   (A+B)_I = A_I + B_I
 
 Here, :math:`X_I` means the value of a coordinate :math:`I` for the tensor 
-:math:`X`. So the value of sum of two tensors is a tensor whose value at a 
-coordinate is the sum of the elements are that coordinate for the two inputs. 
-Unlike many frameworks, it says nothing about storage or arrays.
+:math:`X`. So the value of the sum of two tensors is a tensor whose value at a 
+coordinate is the sum of the elements' two inputs. Unlike many frameworks, it 
```

Comment:
s/the elements' two inputs/the two inputs' element at that coordinate/
s/it/nGraph/